demo/config.js

```diff
@@ -180,6 +180,8 @@ shakaDemo.Config = class {
             'manifest.dash.ignoreEmptyAdaptationSet')
         .addBoolInput_(MessageIds.IGNORE_HLS_TEXT_FAILURES,
             'manifest.hls.ignoreTextStreamFailures')
+        .addBoolInput_(MessageIds.FORCE_HLS_FULL_SEGMENT_REQUEST,
```

Comment:
This should be `USE_FULL_SEGMENTS_FOR_START_TIME`.  If you run `build/all.py`, it should report that `FORCE_HLS_FULL_SEGMENT_REQUEST` is not defined.

Reply:
sorry, missed that while renaming. fixed now